tools/st2_deploy.sh

```diff
@@ -97,6 +99,22 @@ else
     MISTRAL_STABLE_BRANCH="st2-0.5.1"
 fi
 
+update_hosts_file()
+{
+  if [ -z "`cat /etc/hosts | grep -e '127.0.0.1\slocalhost'`" ]; then
```

Comment:
st2api and st2auth are the one causing problem. actually this doesn't have anything to do with the postgresql change.  i can happily remove this and let the f20 fail for now.